android/src/main/kotlin/com/example/native_mixpanel/NativeMixpanelPlugin.kt

```diff
@@ -18,8 +27,14 @@ class NativeMixpanelPlugin: MethodCallHandler {
   override fun onMethodCall(call: MethodCall, result: Result) {
     if (call.method == "getPlatformVersion") {
```

Comment:
@psovit can we remove this call to `getPlatformVersion` ?

Reply:
fixed by removing the call to `getPlatformVersion`